Sito/pagamento.html

```diff
@@ -64,7 +64,7 @@ <h1 lang="ja">Sushi Nakamura</h1>
 	    			<option>Via - n. civico, interno, citta, cap, prov</option>
     			</select>
 
-					<p>Oppure inserisci un nuovo indirizzo:</p>
+					<p>Oppure inseriscine uno:</p>
```

Comment:
Non intendevo cambiare la frase, ma fare in modo che andasse a capo..comunque il problema non era quella frase li quindi puoi rimetterla come prima che si capisce meglio. Il problema deve essere un altro. Non saprei prova a guardare facendo i test (basta che rimpicciolisci il browser e vedi che ad un certo punto il fieldset inizia a fare scroll orizzontale)